jsk_pr2_robot/jsk_pr2_startup/jsk_pr2_lifelog/rosbag_play.launch

```diff
@@ -6,6 +6,10 @@
   <arg name="gui" default="false" />
   <arg name="loop_flag" value="--loop" if="$(arg loop)" />
   <arg name="loop_flag" value="" unless="$(arg loop)" />
+  <arg name="start_time" default="0" />
+  <arg name="duration_time" default="0" />
+  <arg name="duration" default="--duration $(arg duration_time)" if="$(eval duration_time != 0)" />
+  <arg name="duration" default="" if="$(eval duration_time == 0)" />
```

Comment:
Or that implicitly means that JSK should support indigo for other PR2 community?
Of course we should, but even if so, I think few users use indigo client, so I think it's OK. 
(It may be good to write some comments or warnings)